server/commands/accountProvisioner.ts

```diff
@@ -45,6 +45,44 @@ export type AccountProvisionerResult = {
   isNewUser: boolean;
 };
 
+export async function findExistingTeam(
+  authenticationProvider: Props["authenticationProvider"]
+): Promise<TeamCreatorResult | null> {
+  // Should outline deployed in a multi-tenant environment, skip searching
+  // for an existing team.
+  console.log(process.env.DEVELOPMENT);
+  if (process.env.DEPLOYMENT === "hosted") return null;
+
+  // get the first team that exists, ordered by createdAt
+  const team = await Team.findOne({ limit: 1, order: ["createdAt"] });
+  if (team === null) {
+    return null;
+  }
+
+  // query if a corresponding authenticationProvider already exists
+  const authenticationProviders = await team.getAuthenticationProviders({
```

Comment:
This logic doesn't seem to be restricted to OIDC in any way, which means if someone tries to signin with another Google account then it would just remap them to an existing one and allow signin?